.circleci/initialize.py

```diff
@@ -0,0 +1,392 @@
+    )
+
+    params = {"appname": APPNAME}
+
+    result = client_user.execute(query, variable_values=params)
+
+    print("  {APPNAME}".format(APPNAME=APPNAME))
+else:
+    print(result)
+    raise Exception(
+        "\nReview app not initialized"
+        + "\nApp {TARGET_APPNAME} does not exist or the user {USERNAME}".format(
+            TARGET_APPNAME=TARGET_APPNAME, USERNAME=USERNAME
+        )
+        + "does not have permission to query this app.\n"
+    )
+
+if len(linkedServices.items()) != 0:
+    print(
+        "Adding redis/postgres databases...",
+    )
+    for serviceType in linkedServices:
+        if serviceType == "redis":
+            serviceName = "{APPNAME}-{serviceType}".format(
+                APPNAME=APPNAME,
+                serviceType="redis",
+            )[0:30]
+            query_addService = gql(
+                """
+                mutation (
+                    $serviceType: ServiceType=redis,
+                    $serviceName: String
+                ) {
+                    addService (
+                        name: $serviceName,
+                        serviceType: $serviceType
+                    ) {
+                        error
+                    }
+                }
+                """
+            )
+            params_addService = {
+                "serviceName": serviceName,
+                "serviceType": serviceType,
+            }
+            sleep(5)
+            result = client_service.execute(
+                query_addService, variable_values=params_addService
+            )
+
+            print(
+                "  {serviceName}, {serviceType}".format(
+                    serviceName=serviceName, serviceType=serviceType
+                )
+            )
+        elif serviceType == "postgres":
+            serviceName = "{APPNAME}-{serviceType}".format(
+                APPNAME=APPNAME,
+                serviceType="postgres",
+            )[0:30]
+            query_addService = gql(
+                """
+                mutation (
+                    $serviceType: ServiceType=postgres,
+                    $serviceName: String
+                ) {
+                    addService (
+                        name: $serviceName,
+                        serviceType: $serviceType
+                    ) {
+                        error
+                    }
+                }
+                """
+            )
+            params_addService = {
+                "serviceName": serviceName,
+                "serviceType": serviceType,
+            }
+            sleep(5)
+            result = client_service.execute(
+                query_addService, variable_values=params_addService
+            )
+
+            print(
+                "  {serviceName}, {serviceType}".format(
+                    serviceName=serviceName, serviceType=serviceType
+                )
+            )
+else:
+    print("No redis/postgres databases to add")
+
+
+print(
+    "Linking databases...",
```

Comment:
This print statement will still get printed even if there are no databases. Perhaps put this statement in an `if`?

```
if len(linkedService):
    print("Linking databases...")

for serviceType in linkedServices:
    ...
```